code/modules/surgery/organs/ears.dm

```diff
@@ -70,3 +70,24 @@
 	var/obj/item/organ/ears/ears = getorgan(/obj/item/organ/ears)
 	if(ears)
 		ears.minimumDeafTicks(value)
+
+
+/obj/item/organ/ears/cat
+	name = "cat ears"
+	icon = 'icons/obj/clothing/hats.dmi'
+	icon_state = "kitty"
+
+/obj/item/organ/ears/cat/adjustEarDamage(ddmg, ddeaf)
+	..(ddmg*2,ddeaf*2)
```

Comment:
Upsides? Lol no. Downsides? Why not. That's how every other race is. If they don't like it they can choose to not play it. Just like any other race.